#The coordinates of the centroid of the biggest green area in the frame
uint32 centroid_x
uint32 centroid_y

#The biggest area in the frame
float32 area

#The bounding box of the biggest green area in the frame
uint32 bb_x
uint32 bb_y
uint32 bb_width
uint32 bb_height

#The confidence the subscriber should have in this data (from 0 to 100)
uint32 confidence
